### PR TITLE
Allow multiple checksums

### PR DESCRIPTION
**Pull request by mennopruijssers**
uber/ringpop-go#192 adds support for multiple checksums. This PR makes the required changes to the protocol schema.

### CODE REVIEW

File: schema/admin-stats-response.json [hunk 1]
```diff
@@ -115,6 +115,14 @@
                 "checksum": {
                     "type": "number"
                 },
+                "checksums": {
```
Comment (mennopruijssers) [inline schema/admin-stats-response.json]: Since ringpop-node doesn't support it yet, `checksums` is not added to the list with required properties.

Comment (dansimau) [inline schema/admin-stats-response.json]: 👍

File: schema/admin-stats-response.json [hunk 2]
```diff
@@ -115,6 +115,14 @@
...
+                    "type": "object",
+                    "patternProperties": {
+                        "^.*$": {
+                            "type": "number"
```
Comment (dansimau) [inline schema/admin-stats-response.json]: A checksum is always a number?

Comment (mennopruijssers) [inline schema/admin-stats-response.json]: For now yes (and I don't see any reason to change it in the future)

Comment (dansimau) [inline schema/admin-stats-response.json]: The result of a `farmhash` is always a number. That's the bit I was missing (I thought it was alphanumeric). Cool, thanks!

**Pull request merged**